test/sqlancer/mzcompose.py

```diff
@@ -81,18 +81,18 @@ def kill_sqlancer_with_delay() -> None:
         "materialize",
         "--oracle",
         args.oracle,
-        capture=True,
-        capture_stderr=True,
         check=False,
+        detach=True,
```

Comment:
If you detach it, are we certain that it is never going to run past the end of the CI job and just continue running, affecting other jobs that happen to use the same agent in the future? I think it may be wise to force-kill it in a `finally` block, just in case?

Reply:
The `docker compose log -f` should run until the process exits, so we'd only fail in the case of a timeout. I'm not sure if in the case of a timeout we'd still get the chance to kill a process.